src/promptflow-evals/samples/bug-bash/instructions.md

```diff
@@ -0,0 +1,135 @@
+# Promptflow Adversarial Simulator and Evaluator Bug Bash Instructions

```

Comment:
may remove when do the final merge

Reply:
yes, this will be removed after the bugbash.